src/Task/PhpStan/README.md

```diff
@@ -0,0 +1,31 @@
+# phpstan
+
+Check Drupal code against deprecation rules.
+
+### grumphp.yml (with current defaults):
+````yml
+parameters:
+    tasks:
+        phpstan:
+            ignore_patterns: 
+                - '/vendor/'
+                - '/node_modules/'
+                - '/core/'
+                - '/libraries/'
+            extensions: ['php', 'inc', 'module', 'install', 'theme']
+            run_on: ['.']
+            autoload_file:
+              defaults: ~
+              allowed_types: ['string', 'null']
+            configuration:
+              defaults: 'vendor/wunderio/code-quality/config/phpstan.neon'
+              allowed_types: ['string', 'null']
+            memory_limit:
+              defaults: ~
+              allowed_types: ['string', 'null']
+            level:
+              defaults: ~
+              allowed_types: ['string', 'null']
+            extensions:
```

Comment:
Indentation for task extensions incorrect.

Reply:
Fixed